### PR TITLE
KIALI-1721 Overview page: add services view

### PR DESCRIPTION
**Pull request by jotak**
https://issues.jboss.org/browse/KIALI-1721

screenshot

### CODE REVIEW

**Comment by hhovsepy** [comment]
There is some inconsistency between shown services in overview, and in real services list.
Happening to N/A health services.

screenshot from 2018-11-22 17-07-54
screenshot from 2018-11-22 17-08-20
screenshot from 2018-11-22 17-08-40

File: src/pages/Overview/OverviewPage.tsx [hunk 1]
```diff
@@ -176,12 +201,9 @@ class OverviewPage extends React.Component<OverviewProps, State> {
...
-                        <ListPageLink
-                          target={this.state.isApp ? TargetPage.APPLICATIONS : TargetPage.WORKLOADS}
-                          namespace={ns.name}
-                        >
-                          {nbItems === 1 && (this.state.isApp ? '1 Application' : '1 Workload')}
-                          {nbItems !== 1 && nbItems + (this.state.isApp ? ' Applications' : ' Workloads')}
+                        <ListPageLink target={targetPage} namespace={ns.name}>
+                          {nbItems === 1 && oneItemText}
+                          {nbItems !== 1 && nbItems + pluralText}
                         </ListPageLink>
```
Comment (aljesusg) [inline src/pages/Overview/OverviewPage.tsx]: why not 
```js
{nbItems === 1 ? oneItemText : nbItems + pluralText } 
```
?

Comment (jotak) [reply src/pages/Overview/OverviewPage.tsx]: good question, no idea :)
i'll change it

**Comment by jotak** [comment]
Actually, @hhovsepy , I don't reproduce the problem. For instance, on the "default" namespace, I get all three services (docker-registry, kubernetes, router) all green. In details, Envoy health shows "outbound: 1/1" or "Outbound: 3/3".

Or you using Istio 1.0.x or 1.1.x (snapshot) ?

Isn't it a matter of health not fully loaded yet? Like, do you have a loading spinner still spinning while you see "N/A" health in Services list?

**Comment by jotak** [comment]
@aljesusg updated & rebased

**Comment by hhovsepy** [comment]
Verified both PRs.

**Pull request merged**